llvm/unittests/CodeGen/PassManagerTest.cpp

```diff
@@ -202,12 +211,17 @@ TEST_F(PassManagerTest, Basic) {
       TestMachineFunctionPass(Count, Counts)));
   MPM.addPass(TestMachineModulePass(Count, Counts));
   MFPM.addPass(TestMachineFunctionPass(Count, Counts));
+  MFPM.addPass(ReportWarningPass());
   MPM.addPass(createModuleToMachineFunctionPassAdaptor(std::move(MFPM)));
 
+  testing::internal::CaptureStderr();
```

Comment:
TIL `CaptureStderr`, this is good to know

I'd separate the `ReportWarningPass` run out to its own test that doesn't check `Counts`